### PR TITLE
Update update-phone-number-and-email-address.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed the navigation to Correct alternate phone/email address

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3030

File: microsoft-365/admin/manage/update-phone-number-and-email-address.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Use the **Settings and Privacy** page to change your name, mobile phone number,
 
 2. In the header, select your profile icon \> **My account** \> **Settings and Privacy**.
 
-3. In the **Contact Preferences** section, select **Manage and expand Contact preferences**. Update your mobile, phone, and alternate email address. Make sure you use something different from your Microsoft email address for your alternate email address.
+3. In the **Privacy** Tab under  **Contact Preferences** section, select **Contact preferences for admins only**. Update your mobile, phone, and alternate email address. Make sure you use something different from your Microsoft email address for your alternate email address.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/update-phone-number-and-email-address.md]:
```suggestion
3. In the **Privacy** tab under **Contact Preferences**, select **Contact preferences for admins only**. Update your mobile, phone, and alternate email address. Make sure you use something different from your Microsoft email address for your alternate email address.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Pull request merged**